blocks/table/table.css

```diff
@@ -0,0 +1,134 @@
+/* stylelint-disable no-descending-specificity */
+.table-container {
+    margin-bottom: 32px;
+}
+
+.table-container > div.table-wrapper {
+    margin-bottom: 0;
+}
+
+.table-container > .table-wrapper + .table-wrapper {
+    margin-top: 0;
+}
+
+.table-form-submit-wrapper {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+
+.table-form-submit-wrapper button {
+    height: 50px;
+    width: 300px;
+    background-color: var(--blue);
+    color: var(--white);
+    border-radius: 6px;
+    font-family: var(--heading-font-family);
+    font-size: var(--heading-size-s);
+    margin: 20px 0;
+}
+
+.table-form-submit-wrapper button:disabled {
+    background-color: var(--blue-light);
+}
+
+.table-wrapper:not(:has(+ .table-wrapper)) .table tbody tr:last-of-type td {
+    border-bottom: 2px solid var(--blue);
+}
+
+.table {
+    width: 100vw;
+    margin-left: calc(-50vw + 50%);
+}
+
+.table:first-of-type {
+    border-top: 2px solid var(--blue);
```

Comment:
i think this has unintended consequences (esp on small viewport widths). preference to border the table, not the block.

Reply:
Thanks for the feedback, will fix.